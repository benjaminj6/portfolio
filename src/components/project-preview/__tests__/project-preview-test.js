/* eslint-env jest */
import { h } from 'preact'
import { shallow } from 'preact-render-spy'

import ProjectPreview from '../'

test('Social Media CSS Wrapper snap', () => {
  const projects = [
    {
      name: 'My Test Project',
      link: 'www.test.com',
      repo: 'https://github.com/test/test',
      description: 'A cool project'
    }
  ]

  const tree = shallow(<ProjectPreview projects={projects} />)
  expect(tree).toMatchSnapshot()
})
