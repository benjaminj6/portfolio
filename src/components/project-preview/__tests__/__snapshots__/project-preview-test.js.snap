// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social Media CSS Wrapper snap 1`] = `
preact-render-spy (1 nodes)
-------
<div class="jsx-3174525863 jsx-3174525863 ProjectPreview">
  <ul class="jsx-3174525863 jsx-3174525863">
    <li class="jsx-3174525863 jsx-3174525863">
      <Card>
        <h1 class="jsx-3174525863 jsx-3174525863">
          <A
            href="www.test.com"
            target="_blank"
          >
            My Test Project
          </A>
        </h1>
        <p class="jsx-3174525863 jsx-3174525863">A cool project</p>
        <A href="www.test.com">demo</A>
        <A href="https://github.com/test/test">code</A>
      </Card>
    </li>
  </ul>
  <JSXStyle
    styleId="3174525863"
    css="li.jsx-3174525863+li.jsx-3174525863{margin-top:1rem;}p.jsx-3174525863{margin:0.75rem 0;}div.jsx-3174525863 .A+.A{margin-left:1rem;}"
  >
  </JSXStyle>
</div>

`;
